components/eamxx/scripts/machines_specs.py

```diff
@@ -61,11 +61,11 @@
                 "/gpfs/alpine/cli115/proj-shared/scream/master-baselines"),
     "pm-cpu" : ([f"eval $({CIMEROOT}/CIME/Tools/get_case_env -c SMS.ne4pg2_ne4pg2.F2010-SCREAMv1.pm-cpu_gnu)"],
                 ["CC","ftn","cc"],
-                "srun --time 00:30:00 --nodes=1 --constraint=cpu -q regular --account e3sm_g",
+                "salloc --time 00:30:00 --nodes=1 --constraint=cpu -q debug --account e3sm_g",
                 "/global/cfs/cdirs/e3sm/baselines/gnu/scream/pm-cpu"),
-    "pm-gpu" : ([f"eval $({CIMEROOT}/CIME/Tools/get_case_env -c SMS.ne4pg2_ne4pg2.F2010-SCREAMv1.pm-gpu_gnugpu)"],
+    "pm-gpu" : ([f"eval $({CIMEROOT}/CIME/Tools/get_case_env -c SMS.ne4pg2_ne4pg2.F2010-SCREAMv1.pm-gpu_gnugpu)", "echo cuda=true"],
```

Comment:
Hacky trick (`echo cuda=true`) to allow machine_specs to correctly identify this machine as a CUDA machine.